3ds/update_preferences.go

```diff
@@ -0,0 +1,40 @@
+package friends_3ds
+
+import (
+	database_3ds "github.com/PretendoNetwork/friends-secure/database/3ds"
+	"github.com/PretendoNetwork/friends-secure/globals"
+	nex "github.com/PretendoNetwork/nex-go"
+	nexproto "github.com/PretendoNetwork/nex-protocols-go"
+)
+
+func UpdatePreferences(err error, client *nex.Client, callID uint32, showOnline bool, showCurrentGame bool, showPlayedGame bool) {
+	if !showCurrentGame {
+		emptyPresence := nexproto.NewNintendoPresence()
+		emptyPresence.GameKey = nexproto.NewGameKey()
+		emptyPresence.ChangedFlags = 4294967295 // FF FF FF FF, All flags
```

Comment:
Large numbers like this should probbaly be represented as hex for easier ready. `0xFFFFFFFF` instead of 4294967295